templates/oe_webtools/oe-webtools-social-share.html.twig

```diff
@@ -0,0 +1,14 @@
+{#
+/**
+ * @file
+ * Theme override for oe webtools social share block.
+ *
+ * @see ./core/modules/system/templates/block.html.twig
+ */
+#}
+<div class="ecl-social-media-share">
+  <p class="ecl-social-media-share__description">
+      {{ title }}
```

Comment:
Indentation issue. I think it could even go single line like the script tag below.